quickstart/multistage-delivery.sh

```diff
@@ -98,7 +98,17 @@ verify_helm_installation
 print_headline "Downloading demo resources"
 echo "This will create a local folder ./examples"
 echo "git clone https://github.com/keptn/examples --single-branch"
-git clone https://github.com/keptn/examples --single-branch
+
+DIR="./examples"
+BOLD='\033[1m'
+if [ -d "$DIR" ]; then
+  # Example directory already exists
+  echo -e "\n${BOLD}Example directory already exists in folder. Skipping download!"
```

Comment:
I'm not fully sure it this is needed but do you need to disable the bold printing at the end of the line to not have all following outputs also in bold letters?

Reply:
Yes, that is correct. It should be fixed now!